css/css-text-decor/reference/text-decoration-wavy-covers-whole-line-length-001-ref.html

```diff
@@ -0,0 +1,5 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Text Decoration Test: Wavy decorations covers the whole line length mismatch reference file</title>
+<link rel="author" title="Manuel Rego Casasnovas" href="mailto:rego@igalia.com">
+<p>The test passes if you see some green below (the end of a wavy green text decoration).</p>
```

Comment:
np: suggest renaming to -notref.html because this is a mismatch

Reply:
Good suggestion, done.